src/core/or/connection_or.c

```diff
@@ -2448,49 +2474,65 @@ connection_or_send_netinfo,(or_connection_t *conn))
   memset(&cell, 0, sizeof(cell_t));
   cell.command = CELL_NETINFO;
 
+  netinfo_cell_t *netinfo_cell = netinfo_cell_new();
+
   /* Timestamp, if we're a relay. */
   if (public_server_mode(get_options()) || ! conn->is_outgoing)
-    set_uint32(cell.payload, htonl((uint32_t)now));
+    netinfo_cell_set_timestamp(netinfo_cell, (uint32_t)now);
```

Comment:
You sure you can remove the `htonl()` ? Trunnel won't magically switch it :S... Spec specifies:

```
The timestamp is a big-endian unsigned integer number of seconds since the Unix epoch.
```

Reply:
Yes, as trunnel changes the endianness for us:

```
597   /* Encode u32 timestamp */
598   trunnel_assert(written <= avail);
599   if (avail - written < 4)
600     goto truncated;
601   trunnel_set_uint32(ptr, trunnel_htonl(obj->timestamp));
602   written += 4; ptr += 4;
```

Reply:
Great!